src/main/java/frc/robot/utilities/pid/PIDFConfigurable.java

```diff
@@ -7,6 +7,11 @@
     @Override
     PIDFCoefs getCoefs();
 
+    @Override
+    default boolean atSetpoint() {
+        return Math.abs(getError()) <= getTolerance();
+    }
```

Comment:
why do you overide here?

Reply:
Because for the pidf we dont use accelaration therefore there is no deltaTolerance

Reply:
yes we do